web/themes/joinup/joinup_theme.breakpoints.yml

```diff
@@ -0,0 +1,24 @@
+joinup_theme.mobile:
+  label: mobile
+  mediaQuery: ''
+  weight: 0
+  multipliers:
+    - 1x
+joinup_theme.tablet:
+  label: tablet
+  mediaQuery: 'all and (min-width: 768px) and (max-width: 992px)'
```

Comment:
Remarks about breakpoints overlap was made in ticket 2586, please cherry-pick the fix here too.